roles/matrix-backup-borg/defaults/main.yml

```diff
@@ -1,17 +1,17 @@
 ---
 matrix_backup_borg_enabled: true
 
+matrix_backup_borg_base_path: "{{ matrix_base_data_path }}/backup-borg"
+matrix_backup_borg_config_path: "{{ matrix_backup_borg_base_path }}/config"
+
 matrix_backup_borg_container_image_self_build: false
 matrix_backup_borg_docker_repo: "https://github.com/borgmatic-collective/docker-borgmatic"
```

Comment:
Is this correct, by the way?

If self-building, wouldn't we want to self-build https://gitlab.com/etke.cc/borgmatic instead?